### PR TITLE
Add automated tag to A01 and A06 test cases

### PR DESCRIPTION
**Pull request by CathalOConnorRH**
# Description

Added automated tag to test cases for A01 and A06 as there are now automated tests for these. 
## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist
- [ ] This change requires a documentation update 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added a test case that will be used to verify my changes 
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

**Comment by matskiv** [comment]
Thanks @CathalOConnorRH 
I'll approve ones the #756 gets merged, I don't want to accidentally reset retesting on that one, it should be done in the next hour or so.

File: test-cases/tests/installation/a01-verify-that-all-stages-in-the-integreatlyoperator-cr-report.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 estimate: 15m
 tags:
   - happy-path
+  - automated
```
Comment (damienomurchu) [inline test-cases/tests/installation/a01-verify-that-all-stages-in-the-integreatlyoperator-cr-report.md:5]: Can you replace lines 10- 27 with the link to the testcode

File: test-cases/tests/installation/a06-there-are-no-failed-pvcs.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 tags:
   - happy-path
+  - automated
```
Comment (damienomurchu) [inline test-cases/tests/installation/a06-there-are-no-failed-pvcs.md:4]: Can you replace lines 16- 26 with the link to the testcode

**Comment by CathalOConnorRH** [comment]
@damienomurchu Updated those now. Thanks.

**Pull request merged**